ledger/participant-integration-api/src/main/scala/platform/apiserver/services/admin/ApiUserManagementService.scala

```diff
@@ -106,12 +106,11 @@ private[apiserver] final class ApiUserManagementService(
         for {
           _ <- identityProviderExistsOrError(user.identityProviderId)
```

Comment:
Can this be now dropped? Here and for the remaining api methods?

Reply:
actually it cannot be dropped, as we still want to signal the user that `identity_provider_id` which the user provided exists or not. otherwise there will be a foreign key contraint error